<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="TabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>900</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::West</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="home">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(97, 69, 255);
background-color: rgb(16, 5, 95);</string>
    </property>
    <attribute name="title">
     <string>Home</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>475</x>
       <y>90</y>
       <width>350</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>45</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Test Smarter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sign_out">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>840</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(138, 138, 138);</string>
     </property>
     <property name="text">
      <string>Sign Out</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_testing">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(138, 138, 138);</string>
     </property>
     <property name="text">
      <string>Start Testing!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="view_your_tests">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>600</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(138, 138, 138);</string>
     </property>
     <property name="text">
      <string>View Your Tests</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="user_test_table">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>220</y>
       <width>650</width>
       <height>300</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Measurement</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="results">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(97, 69, 255);
background-color: rgb(16, 5, 95);</string>
    </property>
    <attribute name="title">
     <string>Results</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>425</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 3px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
	
	color: rgb(255, 255, 255)</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>425</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 3px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
	
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 3px solid rgb(61, 70, 86);
	color: rgb(255, 255, 255);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 3px solid rgb(43, 50, 61);
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>10</y>
       <width>380</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 3px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
	
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 3px solid rgb(61, 70, 86);
	color: rgb(255, 255, 255);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 3px solid rgb(43, 50, 61);
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>850</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 3px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
	
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 3px solid rgb(61, 70, 86);
	color: rgb(255, 255, 255);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 3px solid rgb(43, 50, 61);
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>55</y>
       <width>380</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 3px solid rgb(52, 59, 72);
	border-radius: 5px;	
	background-color: rgb(52, 59, 72);
	
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 3px solid rgb(61, 70, 86);
	color: rgb(255, 255, 255);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 3px solid rgb(43, 50, 61);
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>850</width>
       <height>730</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>110</y>
       <width>380</width>
       <height>730</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="declineButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>855</y>
       <width>425</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(144, 144, 144);</string>
     </property>
     <property name="text">
      <string>Decline</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submitButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>855</y>
       <width>800</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(144, 144, 144);</string>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="parameters">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(97, 69, 255);
background-color: rgb(16, 5, 95);</string>
    </property>
    <attribute name="title">
     <string>Parameters</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>406</x>
       <y>60</y>
       <width>511</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>37</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Input Test Information</string>
     </property>
    </widget>
    <widget class="QLabel" name="parameter_firstname_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>171</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>180</y>
       <width>500</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="parameter_last_name">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>171</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>260</y>
       <width>500</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="parameters_upload_image">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>350</y>
       <width>171</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(111, 111, 111);</string>
     </property>
     <property name="text">
      <string>Upload Image</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file_path">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>350</y>
       <width>510</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="parameters_image_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>430</y>
       <width>720</width>
       <height>400</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="view_tests">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(97, 69, 255);
background-color: rgb(16, 5, 95);</string>
    </property>
    <attribute name="title">
     <string>View Tests</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header>tabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
